File: src/main/java/in/dragonbra/javasteam/types/UGCHandle.java
package in.dragonbra.javasteam.types;

/**
 * Represents a single unique handle to a piece of User Generated Content.
 */
public class UGCHandle extends GlobalID {

    /**
     * Initializes a new instance of the {@link UGCHandle} class.
     */
    public UGCHandle() {
        super();
    }

    /**
     * Initializes a new instance of the {@link UGCHandle} class.
     *
     * @param ugcId The UGC ID.
     */
    public UGCHandle(long ugcId) {
        super(ugcId);
    }
}

File: src/main/java/in/dragonbra/javasteam/types/UGCHandle.kt
package `in`.dragonbra.javasteam.types

/**
 * Represents a single unique handle to a piece of User Generated Content.
 */
@Suppress("unused")
class UGCHandle : GlobalID {

    /**
     * Initializes a new instance of the [UGCHandle] class.
     */
    constructor() : super()

    /**
     * Initializes a new instance of the [UGCHandle] class.
     * @param ugcId The UGC ID.
     */
    constructor(ugcId: Long) : super(ugcId)
}
